util/cookies.ts

```diff
@@ -18,9 +20,11 @@ export function getParsedCookie(key: string): CookieValue | undefined {
 
 export type CookieValue = {
   id: number;
-  stars: number;
+  note: string;
 }[];
 
+// delete starting here
```

Comment:
- more of these cookie functions should be removed (marked "delete starting here" and ending "delete ending here")
- the `js-cookie` should be removed

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/b185b14ff948fe87a99482d0aa749476c3c23826/util/cookies.ts#L13